### DISCUSSION TITLE
MemoryError: Unable to allocate 1.12 GiB for an array with shape (10, 15000000) and data type float64

### DISCUSSION THREAD

**Discussion by ghost** [vaexio/vaex#1887] [General]
Hey guys,

How i deal with this error?

Also i read the documentation of vaex and find this "Note that this automatic conversion requires free disk space of twice the final HDF5 file size."

I ve got a csv with 140GB, and free space aroun 340 GB. Should be ok right?

My laptop is i7 dell with 8GB ram ( : ( ) and one 1 teragb ssd.

**Comment by maartenbreddels** [comment]
Hi,

> How i deal with this error?

Without knowing what you are trying to do, it's difficult to help. I guess you are trying to read the large cvs? Are you doing the convert=True?

cheers,

Maarten

**Reply by ghost** [reply]
Hi Maarten,

Yes, i am doing convert=true. It worked, i set settings to overcommit memory.
Thanx for answering.
Regards,
John
